clickhouse_sqlalchemy/drivers/http/connector.py

```diff
@@ -30,6 +30,11 @@ class Connection(object):
     transport_cls = RequestsTransport
 
     def __init__(self, *args, **kwargs):
+
+        stream = bool(kwargs.pop('stream', None))
+        self._prefetch = not stream
+
+        # TODO: support `stream` argument in the transport.
```

Comment:
Is this todo still actual?

Reply:
Yes: https://github.com/xzkostyan/clickhouse-sqlalchemy/blob/2f79283d8985fc2877331b72b5f75bfcdf714f92/clickhouse_sqlalchemy/drivers/http/transport.py#L70

But it's not a priority.

The main benefit would be to use e.g. CH's JSON or JSONCompact serialization format in a non-streamed way, for performance reasons.